src/cljc/rems/common/form.cljc

```diff
@@ -1,6 +1,23 @@
 (ns rems.common.form
```

Comment:
should it be rems.common.form-template? or maybe an ns docstring should clarify that this is for form templates only (so far?)